creator-node/src/config.js

```diff
@@ -497,13 +497,13 @@ const config = convict({
     default: 5000 // 5000ms = 5s (prod default)
   },
   enforceWriteQuorum: {
-    doc: 'Boolean flag indicating whether or not primary should reject write on 2/3 replication across replica set',
+    doc: 'Boolean flag indicating whether or not primary should reject write until 2/3 replication across replica set',
     format: Boolean,
     env: 'enforceWriteQuorum',
-    default: false
+    default: true
```

Comment:
i think this is good to just have enabled and use the optimizely flag to control the rollout on prod. stage we would just 100% enable i'd think. so the steps could be 

1. define this optimizely flag and set it to false
2. turn it on for some % of users to observe behavior (and quickly turn off if we notice upload % drop significantly)
3. turn optimizely flag on for everyone
4. remove optimizely flag, default on for everyone

thoughts @SidSethi @theoilie/

Reply:
yeah that looks right

Reply:
so staging can default to true for everyone, but would it make sense to default the env var to false on prod until enough clients actually send the env var? I just checked the client code and didn't see the env var actually defined anywhere so I guess I'd just add it [here](https://github.com/AudiusProject/audius-client/blob/30bbd598d5a9213358c2e4f0d0097c9b97ea2267/packages/web/src/common/services/remote-config/feature-flags.ts) and then pass it as the optimizely value in the routes mentioned in the PR?